charts/rucio-daemons/templates/renew-fts-cronjob.yaml

```diff
@@ -5,14 +5,14 @@
   {{- if or (eq .Values.ftsRenewal.vo "atlas") (eq .Values.ftsRenewal.vo "dteam") }}
   - name: longproxy
     secret:
-      secretName: {{ .Release.Name }}-longproxy
+      secretName: {{ .Values.ftsRenewal.longProxy.existingSecret.name | quote }}
```

Comment:
Because of this change, the update will not be transparent for people who already use the existing charts. Such breaking changes cannot be released outside the major rucio releases (next one in about 2-3 months). In this particular case I'm pretty sure it's possible to change the charts in a way which both introduces the new behavior and is compatible with the old behavior. 

A possible solution would be to set `.Values.ftsRenewal.longProxy.existingSecret.name` to a default empty value. And check here if it's empty. If it's empty, use the old "implicit" name. If it's not empty, use the value. There may be even better ways, I didn't think it through a lot.

Reply:
If researched a bit and found `{{ .Values.ftsRenewal.longProxy.existingSecret.name | default "{{ .Release.Name }}-longproxy" | quote }}` to be the elegant way to do this however the built-in variable `{{ .Release.Name }}` doesn't get picked up that way so I've gone with an if/else instead. I tested it both ways, with an empty value, which results in the usage of the "old" secret name as well as with a different secret name and key, which also works.